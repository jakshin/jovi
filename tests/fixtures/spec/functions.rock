Multiply takes X and Y
Give back X times Y

Put Multiply taking 3, 5, and 9 into Large
Say Large
Say "^ That's 3 * 5 * 9, not (3 * 5) && 9"
