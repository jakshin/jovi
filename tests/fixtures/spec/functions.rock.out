135
^ That's 3 * 5 * 9, not (3 * 5) && 9
